src/schema/v2/artwork/utilities.ts

```diff
@@ -74,12 +74,10 @@ export const embed = (website, { width, height, autoplay }) => {
   }
 }
 
-export const getFigures = ({
-  images,
-  external_video_id,
-  set_video_as_cover,
-  includeAll = false,
-}) => {
+export const getFigures = (
+  { images, external_video_id, set_video_as_cover },
+  { includeAll = false }
+) => {
```

Comment:
The params get passed in the second argument and not in the first 🤦

Reply:
thanks for the quick fix